schemas.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+from typing import List
+
+from pydantic import BaseModel
+
+
+class BaseAuthor(BaseModel):
+    name: str
+    bio: str
+
+class Author(BaseAuthor):
+    id: int
+
+    class Config:
+        from_attributes = True
+
+
+class AuthorCreate(BaseAuthor):
+    pass
+
+
+class BaseBook(BaseModel):
+    title: str
+    summary: str
+    publication_date: datetime
+    authors: List[Author]

```

Comment:
The `authors` field in `BaseBook` should be `author_id: int` instead of `authors: List[Author]`. Each book is associated with a single author, so it should reference an author ID rather than a list of authors.